### PR TITLE
Add the lz4 project to oss-fuzz

### PR DESCRIPTION
**Pull request by cmeister2**
This commit adds support for fuzzers in the lz4 project (https://github.com/lz4/lz4)

It is blocked behind the merge of https://github.com/lz4/lz4/pull/739, but I'm submitting this now as it's worth getting some eyeballs on the proposed options.

Tagging @cyan4973 and @terrelln as they will be the ones receiving all the fun bugs :smile:

### CODE REVIEW

File: projects/lz4/project.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+homepage: "https://github.com/lz4/lz4"
+primary_contact: "Yann.collet.73@gmail.com"
+auto_ccs:
+  - "NickRTerrell@gmail.com"
+  - "cyan@fb.com"
+  - "terrelln@fb.com"
+sanitizers:
```
Comment (terrelln) [inline projects/lz4/project.yaml]: Please add the `memory` sanitizer. Lz4 should be MSAN clean.

File: projects/lz4/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2016 Google Inc.
```
Comment (jonathanmetzman) [inline projects/lz4/Dockerfile]: nit: 2019.

File: projects/lz4/build.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -eu
+# Copyright 2016 Google Inc.
```
Comment (jonathanmetzman) [inline projects/lz4/build.sh]: nit: 2019

**Comment by jonathanmetzman** [comment]
We approve of adding this project.

File: projects/lz4/project.yaml (hunk 1 above)
Comment (cmeister2) [reply projects/lz4/project.yaml]: Fixed.

File: projects/lz4/Dockerfile (hunk 2 above)
Comment (cmeister2) [reply projects/lz4/Dockerfile]: Fixed

File: projects/lz4/build.sh (hunk 3 above)
Comment (cmeister2) [reply projects/lz4/build.sh]: Fixed.

**Comment by cmeister2** [comment]
@jonathanmetzman CI has now passed as the code is now in the lz4 project.

**Pull request merged**